README.md

```diff
@@ -677,64 +677,23 @@ added 1 package from 1 contributor and audited 8438 packages in 4.257s
 
 <br />
 
-### Step 8: Rename Layout Template File from `app.html.eex` to `app.html.leex`
+### Step 8: Add the CSRF meta tag to the App Layout
```

Comment:
I mostly reverted this back to https://github.com/dwyl/phoenix-liveview-counter-tutorial/pull/6/files